extension-manager-integration-test-java/src/test/java/com/exasol/extensionmanager/itest/ExasolVersionCheckIT.java

```diff
@@ -0,0 +1,32 @@
+package com.exasol.extensionmanager.itest;
```

Comment:
Does this test set up an Exasol docker database in order to execute an integration test that verifies the version check feature?